app/adapter/graphql/resolver/error_test.go

```diff
@@ -0,0 +1,154 @@
+package resolver
```

Comment:
Let's remove this file.

The master of Go said: "As an aside, I believe you should never inspect the output of the error. Error method. The Error method on the error interface exists for humans, not code."

https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully